lib/tarantool_server.py

```diff
@@ -129,6 +136,7 @@ def killall_servers(self, server, ts, crash_occured):
 
     def execute(self, server):
         server.current_test = self
+        server.pre_cleanup()
```

Comment:
Finally I understood we should not remove xlogs for working tarantool server. But we should do it for non-default servers and app server.